### ISSUE TITLE
Google shows weird results when you search for covidactnow.org.

### ISSUE THREAD

**Issue by mikelehen** [covid-projections/covid-projections#411]
Google for e.g. `covidactnow.org`:
![image](https://user-images.githubusercontent.com/206364/78419072-b3f7fd80-75f6-11ea-8fe2-600244f3f751.png)

Not sure what all we need to do to fix this.  From first glance though I notice that we're populating meta title/description tags on the model pages (see https://github.com/covid-projections/covid-projections/blob/ddf1929c4b58170c696da5a437f18701f273adc2/src/screens/ModelPage/ModelPage.js#L325), but not on the HomePage, FAQ, etc. Doing so would probably help?

@szhu mentioned a sitemap may help as well. https://support.google.com/webmasters/answer/183668?hl=en

**Comment by szhu** [comment]
A few observations:
- Google calls these things "Sitelinks". https://support.google.com/webmasters/answer/47334
- Currently, all pages on the site have have the statically generated title "Covid Act Now", yet they have distinct titles in as sitelinks. So we can probably fix this without changing any of the HTML. (cc #284, issue on server side rendering)
- The sitelink titles that say "You must act now", "Keep staying at home" are probably taken from the document title that's set by the React app.
- The sitelink titles that say "California", "Virginia"… where are these from? I have two guesses:
  - A lot of websites have the website name in every page title, so Google is probably factoring them out and keeping just the text that changes between every page… leaving only the state names.
  - People are linking to these pages with the state names, and Google is using these.
  - It's inferring it from the URL?

Based on this, I think a good shot at solving this would be to update the title set in `ModelPage.js` to be of the form "{state name} • Covid Act Now". Google will consistently factor out "Covid Act Now" and leave only the state names.

**Comment by szhu** [comment]
Another observation: The snippets that are shown under the sitelinks are hard to control, even for major sites.

| ![image](https://user-images.githubusercontent.com/1570168/78423843-0e588480-761e-11ea-994c-a9ce8ea88758.png) | ![image](https://user-images.githubusercontent.com/1570168/78423846-13b5cf00-761e-11ea-8a3b-6e72cb48422f.png) |
| --- | --- |

**Comment by szhu** [comment]
Discussed with @maxwhenderson. Given that we can't change which pages appear as sitelinks and the order in which they appear, we decided that this is a reasonable target of what we'd like the search result descriptions to look like:

| Before | After |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/1570168/78424265-28479680-7621-11ea-871d-bc86f628d347.png) | ![image](https://user-images.githubusercontent.com/1570168/78424249-04845080-7621-11ea-9732-c4b8f8877ea3.png) |

Open questions I still have:
- What do we want the homepage description to be? There is currently none set in `index.html`. Here is the title/description we use for sharing
  - "This model predicts the last day each state can act before the point of no return"
  - "The only thing that matters right now is the speed of your response"
- The descriptions are fairly vague.
  - There is no target audience. Whose response are we talking about? Previously there was text that said "Public officials:" -- I think that was useful. But also, a narrative can become hard to follow if the POV keeps changing throughout the website. If the page's target audience is everyone, then a message like "The only thing that matters is the how quickly your public officials respond" be less jarring.
  - All of the titles and descriptions make no explicit references to COVID-19 or Coronavirus. It seems… almost intentionally vague? Not sure if we're leaving it out for some reason. The domain and title of the site contain "Covid" but it's capitalized differently and not referenced directly either. Also, until you click into the site, it's not clear what kind of action "act" is implying.
- The word "model" might only have meaning to the stats/data-science community, not politicians or the general public.

Here's my best attempt at resolving the open questions above:

![image](https://user-images.githubusercontent.com/1570168/78424902-7c547a00-7625-11ea-9f15-4ca634e764d7.png)


<details><summary>Addendum: the html I used so I can use it for later:</summary>

```
<div><!--m--><link href="https://covidactnow.org/" rel="prerender"><div class="rc" data-hveid="CAUQAQ" data-ved="2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQFSgAMAB6BAgFEAE"><div class="r"><a href="https://covidactnow.org/" ping="/url?sa=t&amp;source=web&amp;rct=j&amp;url=https://covidactnow.org/&amp;ved=2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQFjAAegQIBRAC&amp;authuser=1"><br><h3 class="LC20lb DKV0Md">Covid Act Now</h3><div class="TbwUpd NJjxre"><cite class="iUh30 tjvcx">covidactnow.org</cite></div></a><div class="B6fmyf"><div class="TbwUpd"><cite class="iUh30 tjvcx">covidactnow.org</cite></div><div class="eFM0qc"><span><div class="action-menu"><a class="GHDvEf" href="#" id="am-b0" aria-label="Result Options" aria-expanded="false" aria-haspopup="true" role="button" jsaction="m.tdd;keydown:m.hbke;keypress:m.mskpe" data-ved="2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQ7B0wAHoECAUQAw"><span class="mn-dwn-arw"></span></a><ol class="action-menu-panel" role="menu" tabindex="-1" jsaction="keydown:m.hdke;mouseover:m.hdhne;mouseout:m.hdhue" data-ved="2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQqR8wAHoECAUQBA"><li class="action-menu-item" role="menuitem"><a class="fl" href="https://webcache.googleusercontent.com/search?q=cache:Xx3nJRWXH80J:https://covidactnow.org/+&amp;cd=1&amp;hl=en&amp;ct=clnk&amp;gl=us" ping="/url?sa=t&amp;source=web&amp;rct=j&amp;url=https://webcache.googleusercontent.com/search%3Fq%3Dcache:Xx3nJRWXH80J:https://covidactnow.org/%2B%26cd%3D1%26hl%3Den%26ct%3Dclnk%26gl%3Dus&amp;ved=2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQIDAAegQIBRAF&amp;authuser=1">Cached</a></li></ol></div></span></div></div></div><div class="s">This model predicts the last day each state can act before the point of no return.<br></div></div><!--n--><table class="nrgt" cellpadding="0" cellspacing="0"><tbody><tr class="mslg dmenKe"><td><!--m--><div class="sld vsc"><span class="cNifBc"><h3 class="r"><a class="l" href="https://www.covidactnow.org/state/CA" data-ved="2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQjBAwAXoECAUQCA" ping="/url?sa=t&amp;source=web&amp;rct=j&amp;url=https://www.covidactnow.org/state/CA&amp;ved=2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQjBAwAXoECAUQCA&amp;authuser=1">California Forecast</a></h3></span><div class="s"><div class="st" style="overflow:hidden;width:220px">Keep staying at home in California.<br></div></div></div><!--n--></td><td><!--m--><div class="sld vsc"><span class="cNifBc"><h3 class="r"><a class="l" href="https://covidactnow.org/state/HI" data-ved="2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQjBAwBnoECAUQCg" ping="/url?sa=t&amp;source=web&amp;rct=j&amp;url=https://covidactnow.org/state/HI&amp;ved=2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQjBAwBnoECAUQCg&amp;authuser=1">Hawaii Forecast</a></h3></span><div class="s"><div class="st" style="overflow:hidden;width:220px"><div class="st" style="overflow:hidden;width:220px">Keep staying at home in Hawaii.</div></div></div></div><!--n--></td></tr><tr class="mslg"><td><!--m--><div class="sld vsc"><span class="cNifBc"><h3 class="r"><a class="l" href="https://covidactnow.org/state/TX" data-ved="2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQjBAwAnoECAUQDA" ping="/url?sa=t&amp;source=web&amp;rct=j&amp;url=https://covidactnow.org/state/TX&amp;ved=2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQjBAwAnoECAUQDA&amp;authuser=1">Texas Forecast</a></h3></span><div class="s"><div class="st" style="overflow:hidden;width:220px"><div class="st" style="overflow:hidden;width:220px">Keep staying at home in Texas.</div></div></div></div><!--n--></td><td><!--m--><div class="sld vsc"><span class="cNifBc"><h3 class="r"><a class="l" href="https://covidactnow.org/about" data-ved="2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQjBAwB3oECAUQDg" ping="/url?sa=t&amp;source=web&amp;rct=j&amp;url=https://covidactnow.org/about&amp;ved=2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQjBAwB3oECAUQDg&amp;authuser=1">About</a></h3></span><div class="s"><div class="st" style="overflow:hidden;width:220px">CovidActNow.org was created by a team of data ...<br></div></div></div><!--n--></td></tr><tr class="mslg"><td><!--m--><div class="sld vsc"><span class="cNifBc"><h3 class="r"><a class="l" href="https://covidactnow.org/state/FL" data-ved="2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQjBAwA3oECAUQEA" ping="/url?sa=t&amp;source=web&amp;rct=j&amp;url=https://covidactnow.org/state/FL&amp;ved=2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQjBAwA3oECAUQEA&amp;authuser=1">Florida Forecast</a></h3></span><div class="s"><div class="st" style="overflow:hidden;width:220px"><div class="st" style="overflow:hidden;width:220px">Keep staying at home in Florida.</div></div></div></div><!--n--></td><td><!--m--><div class="sld vsc"><span class="cNifBc"><h3 class="r"><a class="l" href="https://covidactnow.org/model" data-ved="2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQjBAwCHoECAUQEg" ping="/url?sa=t&amp;source=web&amp;rct=j&amp;url=https://covidactnow.org/model&amp;ved=2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQjBAwCHoECAUQEg&amp;authuser=1">Model</a></h3></span><div class="s"><div class="st" style="overflow:hidden;width:220px">The U.S. Intervention Model is a data platform that projects ...<br></div></div></div><!--n--></td></tr><tr class="mslg"><td><!--m--><div class="sld vsc"><span class="cNifBc"><h3 class="r"><a class="l" href="https://covidactnow.org/state/SD" data-ved="2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQjBAwBHoECAUQFA" ping="/url?sa=t&amp;source=web&amp;rct=j&amp;url=https://covidactnow.org/state/SD&amp;ved=2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQjBAwBHoECAUQFA&amp;authuser=1">South Dakota Forecast</a></h3></span><div class="s"><div class="st" style="overflow:hidden;width:220px"><div class="st" style="overflow:hidden;width:220px">You must act now in South Dakota.</div></div></div></div><!--n--></td><td><!--m--><div class="sld vsc"><span class="cNifBc"><h3 class="r"><a class="l" href="https://covidactnow.org/endorsements" data-ved="2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQjBAwCXoECAUQFg" ping="/url?sa=t&amp;source=web&amp;rct=j&amp;url=https://covidactnow.org/endorsements&amp;ved=2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQjBAwCXoECAUQFg&amp;authuser=1">Endorsements</a></h3></span><div class="s"><div class="st" style="overflow:hidden;width:220px">While no projection is perfect, we endorse this tool and model ...</div></div></div><!--n--></td></tr><tr class="mslg"><td><!--m--><div class="sld vsc"><span class="cNifBc"><h3 class="r"><a class="l" href="https://covidactnow.org/state/LA" data-ved="2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQjBAwBXoECAUQGA" ping="/url?sa=t&amp;source=web&amp;rct=j&amp;url=https://covidactnow.org/state/LA&amp;ved=2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQjBAwBXoECAUQGA&amp;authuser=1">Louisiana Forecast</a></h3></span><div class="s"><div class="st" style="overflow:hidden;width:220px">Keep staying at home in Louisiana.</div></div></div><!--n--></td><td><!--m--><div class="sld vsc"><span class="cNifBc"><h3 class="r"><a class="l" href="https://covidactnow.org/state/VA" data-ved="2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQjBAwCnoECAUQGg" ping="/url?sa=t&amp;source=web&amp;rct=j&amp;url=https://covidactnow.org/state/VA&amp;ved=2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQjBAwCnoECAUQGg&amp;authuser=1">Virginia Forecast</a></h3></span><div class="s"><div class="st" style="overflow:hidden;width:220px">Keep staying at home in Virginia.</div></div></div><!--n--></td></tr><tr><td colspan="2" style="padding-left:6px;vertical-align:top"><div class="mrf"><a class="fl" href="/search?authuser=1&amp;sxsrf=ALeKk03eF3wmkMzrvMiL0cTpLhv-RcEiuw:1585993972462&amp;q=+site:covidactnow.org+covidactnow.org&amp;sa=X&amp;ved=2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQrAN6BAgFEBs">More results from covidactnow.org&nbsp;»</a></div></td></tr></tbody></table></div>
```

```
<div><!--m--><link href="https://covidactnow.org/" rel="prerender"><div class="rc" data-hveid="CAUQAQ" data-ved="2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQFSgAMAB6BAgFEAE"><div class="r"><a href="https://covidactnow.org/" ping="/url?sa=t&amp;source=web&amp;rct=j&amp;url=https://covidactnow.org/&amp;ved=2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQFjAAegQIBRAC&amp;authuser=1"><br><h3 class="LC20lb DKV0Md">Covid Act Now</h3><div class="TbwUpd NJjxre"><cite class="iUh30 tjvcx">covidactnow.org</cite></div></a><div class="B6fmyf"><div class="TbwUpd"><cite class="iUh30 tjvcx">covidactnow.org</cite></div><div class="eFM0qc"><span><div class="action-menu"><a class="GHDvEf" href="#" id="am-b0" aria-label="Result Options" aria-expanded="false" aria-haspopup="true" role="button" jsaction="m.tdd;keydown:m.hbke;keypress:m.mskpe" data-ved="2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQ7B0wAHoECAUQAw"><span class="mn-dwn-arw"></span></a><ol class="action-menu-panel" role="menu" tabindex="-1" jsaction="keydown:m.hdke;mouseover:m.hdhne;mouseout:m.hdhue" data-ved="2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQqR8wAHoECAUQBA"><li class="action-menu-item" role="menuitem"><a class="fl" href="https://webcache.googleusercontent.com/search?q=cache:Xx3nJRWXH80J:https://covidactnow.org/+&amp;cd=1&amp;hl=en&amp;ct=clnk&amp;gl=us" ping="/url?sa=t&amp;source=web&amp;rct=j&amp;url=https://webcache.googleusercontent.com/search%3Fq%3Dcache:Xx3nJRWXH80J:https://covidactnow.org/%2B%26cd%3D1%26hl%3Den%26ct%3Dclnk%26gl%3Dus&amp;ved=2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQIDAAegQIBRAF&amp;authuser=1">Cached</a></li></ol></div></span></div></div></div><div class="s">Urge your public officials to quickly take action against the COVID-19 pandemic. These charts predict the last day each state can act before the point of no return.<br></div></div><!--n--><table class="nrgt" cellpadding="0" cellspacing="0"><tbody><tr class="mslg dmenKe"><td><!--m--><div class="sld vsc"><span class="cNifBc"><h3 class="r"><a class="l" href="https://www.covidactnow.org/state/CA" data-ved="2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQjBAwAXoECAUQCA" ping="/url?sa=t&amp;source=web&amp;rct=j&amp;url=https://www.covidactnow.org/state/CA&amp;ved=2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQjBAwAXoECAUQCA&amp;authuser=1">California Forecast</a></h3></span><div class="s"><div class="st" style="overflow:hidden;width:220px">California: Keep staying at home to protect against the COVID-19 …<br></div></div></div><!--n--></td><td><!--m--><div class="sld vsc"><span class="cNifBc"><h3 class="r"><a class="l" href="https://covidactnow.org/state/HI" data-ved="2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQjBAwBnoECAUQCg" ping="/url?sa=t&amp;source=web&amp;rct=j&amp;url=https://covidactnow.org/state/HI&amp;ved=2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQjBAwBnoECAUQCg&amp;authuser=1">Hawaii Forecast</a></h3></span><div class="s"><div class="st" style="overflow:hidden;width:220px"><div class="st" style="overflow:hidden;width:220px">Hawaii: Keep staying at home in to protect against the COVID-19 …</div></div></div></div><!--n--></td></tr><tr class="mslg"><td><!--m--><div class="sld vsc"><span class="cNifBc"><h3 class="r"><a class="l" href="https://covidactnow.org/state/TX" data-ved="2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQjBAwAnoECAUQDA" ping="/url?sa=t&amp;source=web&amp;rct=j&amp;url=https://covidactnow.org/state/TX&amp;ved=2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQjBAwAnoECAUQDA&amp;authuser=1">Texas Forecast</a></h3></span><div class="s"><div class="st" style="overflow:hidden;width:220px"><div class="st" style="overflow:hidden;width:220px">Texas: Keep staying at home to protect against the COVID-19 …</div></div></div></div><!--n--></td><td><!--m--><div class="sld vsc"><span class="cNifBc"><h3 class="r"><a class="l" href="https://covidactnow.org/about" data-ved="2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQjBAwB3oECAUQDg" ping="/url?sa=t&amp;source=web&amp;rct=j&amp;url=https://covidactnow.org/about&amp;ved=2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQjBAwB3oECAUQDg&amp;authuser=1">About</a></h3></span><div class="s"><div class="st" style="overflow:hidden;width:220px">CovidActNow.org was created by a team of data ...<br></div></div></div><!--n--></td></tr><tr class="mslg"><td><!--m--><div class="sld vsc"><span class="cNifBc"><h3 class="r"><a class="l" href="https://covidactnow.org/state/FL" data-ved="2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQjBAwA3oECAUQEA" ping="/url?sa=t&amp;source=web&amp;rct=j&amp;url=https://covidactnow.org/state/FL&amp;ved=2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQjBAwA3oECAUQEA&amp;authuser=1">Florida Forecast</a></h3></span><div class="s"><div class="st" style="overflow:hidden;width:220px"><div class="st" style="overflow:hidden;width:220px">Florida: Keep staying at home to protect against the COVID-19 …<br></div></div></div></div><!--n--></td><td><!--m--><div class="sld vsc"><span class="cNifBc"><h3 class="r"><a class="l" href="https://covidactnow.org/model" data-ved="2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQjBAwCHoECAUQEg" ping="/url?sa=t&amp;source=web&amp;rct=j&amp;url=https://covidactnow.org/model&amp;ved=2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQjBAwCHoECAUQEg&amp;authuser=1">Model</a></h3></span><div class="s"><div class="st" style="overflow:hidden;width:220px">The U.S. Intervention Model is a data platform that projects ...<br></div></div></div><!--n--></td></tr><tr class="mslg"><td><!--m--><div class="sld vsc"><span class="cNifBc"><h3 class="r"><a class="l" href="https://covidactnow.org/state/SD" data-ved="2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQjBAwBHoECAUQFA" ping="/url?sa=t&amp;source=web&amp;rct=j&amp;url=https://covidactnow.org/state/SD&amp;ved=2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQjBAwBHoECAUQFA&amp;authuser=1">South Dakota Forecast</a></h3></span><div class="s"><div class="st" style="overflow:hidden;width:220px"><div class="st" style="overflow:hidden;width:220px">South Dakota: Urge your public officials to act now against the …</div></div></div></div><!--n--></td><td><!--m--><div class="sld vsc"><span class="cNifBc"><h3 class="r"><a class="l" href="https://covidactnow.org/endorsements" data-ved="2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQjBAwCXoECAUQFg" ping="/url?sa=t&amp;source=web&amp;rct=j&amp;url=https://covidactnow.org/endorsements&amp;ved=2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQjBAwCXoECAUQFg&amp;authuser=1">Endorsements</a></h3></span><div class="s"><div class="st" style="overflow:hidden;width:220px">While no projection is perfect, we endorse this tool and model ...</div></div></div><!--n--></td></tr><tr class="mslg"><td><!--m--><div class="sld vsc"><span class="cNifBc"><h3 class="r"><a class="l" href="https://covidactnow.org/state/LA" data-ved="2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQjBAwBXoECAUQGA" ping="/url?sa=t&amp;source=web&amp;rct=j&amp;url=https://covidactnow.org/state/LA&amp;ved=2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQjBAwBXoECAUQGA&amp;authuser=1">Louisiana Forecast</a></h3></span><div class="s"><div class="st" style="overflow:hidden;width:220px">Louisiana: Keep staying at home to protect against the COVID-19 …</div></div></div><!--n--></td><td><!--m--><div class="sld vsc"><span class="cNifBc"><h3 class="r"><a class="l" href="https://covidactnow.org/state/VA" data-ved="2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQjBAwCnoECAUQGg" ping="/url?sa=t&amp;source=web&amp;rct=j&amp;url=https://covidactnow.org/state/VA&amp;ved=2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQjBAwCnoECAUQGg&amp;authuser=1">Virginia Forecast</a></h3></span><div class="s"><div class="st" style="overflow:hidden;width:220px">Virginia: Keep staying at home to protect against the COVID-19 …</div></div></div><!--n--></td></tr><tr><td colspan="2" style="padding-left:6px;vertical-align:top"><div class="mrf"><a class="fl" href="/search?authuser=1&amp;sxsrf=ALeKk03eF3wmkMzrvMiL0cTpLhv-RcEiuw:1585993972462&amp;q=+site:covidactnow.org+covidactnow.org&amp;sa=X&amp;ved=2ahUKEwiptuD6v87oAhWjOn0KHVaLA_gQrAN6BAgFEBs">More results from covidactnow.org&nbsp;»</a></div></td></tr></tbody></table></div>
```
</details>

**Comment by jamestamplin** [comment]
cc @Jtensminger as FYI

**Comment by szhu** [comment]
Made a further discovery while working on solving this: I don't think any of the page-specific Facebook descriptions are working, since (1) Facebook doesn't interpret `<meta>` tags in JavaScript and (2) 

Tested with the following methods:
1. <https://developers.facebook.com/tools/debug/?q=https%3A%2F%2Fcovidactnow.org%2Fus%2Fca>
2. Went to <https://covidactnow.org/us/ca> and clicked the FB share button
3. Sent myself <https://covidactnow.org/us/ca> in Facebook Messenger

With all 3 results, only the title and description for the home page was returned.
